src/commands/nervous_system_parameters.rs

```diff
@@ -0,0 +1,18 @@
+use crate::lib::signing::{sign_query_as_ingress_with_request_id, IngressWithRequestId};
+use crate::lib::{AnyhowResult, TargetCanister};
+use crate::SnsCanisterIds;
+use candid::Encode;
+use ic_base_types::PrincipalId;
+
+pub fn exec(sns_canister_ids: &SnsCanisterIds) -> AnyhowResult<Vec<IngressWithRequestId>> {
```

Comment:
Please add a `///` style comment at the top of this method (you can copy the one you write in src/commands/mod.rs. Each command written so far has kept with this convention and I think we should continue that.